person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :is_of_age?
```

Comment:
- [OPTIONAL] there is a small issue here, The name of the method should be the same 👍  
> I am sure that you will fix this minor issue, so I marked it as optional

Reply:
@Bondok6 Yeah sure I will fix it. Thanks for your approval